api/src/opentrons/protocol_engine/state/_labware_origin_math.py

```diff
@@ -441,6 +466,18 @@ def _parent_origin_to_slot_back_left_bottom(
     return Point(x, y, z)
 
 
+def _flex_tip_rack_lid_feature_to_child_origin(
+    child_labware: LabwareDefinition3,
+) -> Point:
+    """Returns the offset from a parent lid's Flex tip rack lid locating feature to the child origin."""
```

Comment:
Should this be

```suggestion
    """Returns the offset from a child lid's Flex tip rack lid locating feature to the child origin."""
```

(Parent tip rack on bottom, child lid on top)